src/api/app/helpers/webui/notification_helper.rb

```diff
@@ -1,11 +1,68 @@
+# rubocop:disable Metrics/ModuleLength
 module Webui::NotificationHelper
+  TRUNCATION_LENGTH = 100
+  TRUNCATION_ELLIPSIS_LENGTH = 3 # `...` is the default ellipsis for String#truncate
```

Comment:
We're [duplicating code](https://github.com/openSUSE/open-build-service/blob/9081ff565bd2afa5af48bf179e23f12f12f9f665/src/api/app/helpers/webui/notification_excerpt_helper.rb#L2)...

Reply:
I introduced another commit removing the NotificationExcerptHelper. This code is moved from there to NotificationHelper.